tests/tools/test_ontoconvert.py

```diff
@@ -39,3 +39,18 @@ def test_run() -> None:
     assert re.search("@prefix : <https://w3id.org/ex/testonto#>", output2)
     assert re.search("<https://w3id.org/ex/testonto> .* owl:Ontology", output2)
     assert re.search("testclass .* owl:Class", output2)
+
+    # Test 3 - copy-annotation
```

Comment:
you are checking copy_preflabel. I agree that this is an alias for annotation, but I think it would be good to check that other annotations also are added. Most particularly, that we can copy to an  annotation that is originally not in the ontology.

Also, I think we should have a test where preflabel is not converted, as prefLabel is copied by default (if I understand store_true correctly).